src/github-api.ts

```diff
@@ -54,9 +61,23 @@ export default class GithubAPI {
     return this._fetch(`${prefix}/repos/${repo}/issues/${issue}`);
   }
 
-  public async getUserData(login: string): Promise<GitHubUserResponse> {
+  public async getUserData(userInfo: Pick<GithubUserInfo, "login" | "html_url">): Promise<GitHubContributor> {
+    let login = userInfo.login;
+    let path = "users";
+
+    if (userInfo.html_url && userInfo.html_url.includes("/apps/")) {
+      path = "apps";
+      login = userInfo.html_url.split("/").pop() as string;
+    }
     const prefix = process.env.GITHUB_API_URL || `https://api.${this.github}`;
-    return this._fetch(`${prefix}/users/${login}`);
+    const data = await this._fetch(`${prefix}/${path}/${login}`);
+
+    if (login === "Copilot") {
```

Comment:
I think this should be at a different level, maybe where we ignore committers?

Reply:
I think the sanitization should go at the same level as the ignoring, is what I mean